diff --git a/node_modules/@capacitor-community/admob/android/build.gradle b/node_modules/@capacitor-community/admob/android/build.gradle
index 1056a18..4ed1d7f 100644
--- a/node_modules/@capacitor-community/admob/android/build.gradle
+++ b/node_modules/@capacitor-community/admob/android/build.gradle
@@ -37,7 +37,8 @@ android {
     buildTypes {
         release {
             minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+            //proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt')
         }
     }
     lintOptions {
